content/docs/0.19.x/manage/plan/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Plan your project
+description: Create and delete a project in Keptn.
+weight: 10
+keywords: [0.16.x-manage]
+aliases:
+---
+
+It is useful to think about what you want your Keptn project to do
+before you begin implementing it.
+This page summarizes some things you should consider.
+You can then use the [Create project](../project) page
+for instructions about implementing a project.
+
+It is important to understand that a Keptn project is **not** a pipeline.
+Instead, activities are triggered by specific events.
+For a broader explanation of this, you can read the seminal
+[Continuous Delivery without pipelines -- How it works and why you need it](https://medium.com/keptn/continuous-delivery-without-pipelines-7e84db8c8261) article.
+
+## What should this project do?
+
+Before you can run the Keptn command to create a project,
+you must populate a [shipyard.yaml](../../reference/files/shipyard) file
+that defines the `stages` for your project.
```

Comment:
defines stages and sequences of tasks that your keptn project will be able to orchestrate

Reply:
I did a bit of rewording -- see what you think.